test-suite-dockerised/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+plugins {
+    groovy
+    id("io.micronaut.build.internal.azure-example")
+    id("com.microsoft.azure.azurefunctions") version "1.15.0"
+}
+
+version = "1.0"
+
+micronaut {
+    version("4.3.5")
```

Comment:
can you define a platform version in  libs.version.toml  and use that. Same for the azure functions gradle plugin version

Reply:
https://github.com/micronaut-projects/micronaut-azure/pull/650/commits/86105e112a3a9ce9e83e2fe12032ee67ad34c1f4